src/translations/en.json

```diff
@@ -5571,10 +5571,16 @@
         },
         "core-config": {
           "intro": "Hello {name}, welcome to Home Assistant. How would you like to name your home?",
+          "intro_core": "We will be setting up some core configuration for your Home Assistant instance now, you can always change this later in the configuration panel.",
           "intro_location": "We would like to know where you live. This information will help with displaying information and setting up sun-based automations. This data is never shared outside of your network.",
```

Comment:
```suggestion
          "intro_location": "If you tell us where you live, we can show location-based information such as the weather and set up sun-based automations. This data is never shared outside of your network.",
```

Reply:
_We would like to know where you live_ sounds a bit creepy to me.

Reply:
The "if" sentence make me think it's a "optional" step but this step is mandatory. Do you think it can cause issue?